src/test/java/ru/practicum/shareit/controller/BookingControllerTest.java

```diff
@@ -0,0 +1,247 @@
+package ru.practicum.shareit.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import ru.practicum.shareit.booking.BookingController;
+import ru.practicum.shareit.booking.BookingService;
+import ru.practicum.shareit.booking.Status;
+import ru.practicum.shareit.booking.dto.BookingDto;
+import ru.practicum.shareit.booking.dto.BookingResponseDto;
+import ru.practicum.shareit.exception.CannotBookItemException;
+import ru.practicum.shareit.exception.NotFoundException;
+
+import ru.practicum.shareit.item.model.Item;
+
+import ru.practicum.shareit.user.model.User;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(controllers = BookingController.class)
+@AutoConfigureMockMvc
+@RequiredArgsConstructor(onConstructor_ = @Autowired)
+public class BookingControllerTest {
+
+    private final MockMvc mockMvc;
+
+    private final ObjectMapper objectMapper;
+
+    private User booker;
+    private User owner;
+
+    private Item item;
+    private BookingDto bookingDto;
+    private BookingResponseDto bookingResponseDto;
+    LocalDateTime start;
+    LocalDateTime end;
+    public static final String USER_ID_HEADER = "X-Sharer-User-Id";
+
+    @MockBean
+    private BookingService bookingService;
+
+    @BeforeEach
+    public void setUp() throws Exception {
```

Comment:
Метод не выбрасывает ошибок
```suggestion
    public void setUp() {
```